apps/frontend/src/app/pages/company/applications-profile/applications-profile.component.html

```diff
@@ -0,0 +1,142 @@
+<div class="container mx-auto min-h-screen px-3">
+  <div class="md:flex items-start gap-40">
+    <div
+      class="flex gap-2 items-center py-8 cursor-pointer"
+      [routerLink]="['/']"
+    >
+      <i class="bi bi-arrow-left text-base-100 text-xl"></i>
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/ed9d1983-8cc8-497c-b627-bc5ef73d2c49)

If the picture was closer to the edge of the screen the way the text is, it would look much better